sql/report_util_functions.sql

```diff
@@ -723,6 +723,31 @@ BEGIN
 END$$
 DELIMITER ;
 
+-- patientReasonForConsultationIsUnplannedAid
+
+DROP FUNCTION IF EXISTS patientReasonForConsultationIsUnplannedAid;
+
+DELIMITER $$
+CREATE FUNCTION patientReasonForConsultationIsUnplannedAid(
+    p_patientId INT(11)) RETURNS TINYINT(1)
+    DETERMINISTIC
+BEGIN 
+    DECLARE patientIsUnplannedAid TINYINT(1) DEFAULT 0;
+
+    DECLARE uuidPatientIsUnplannedAid VARCHAR(38) DEFAULT "17a3b24b-e107-49fe-8b0d-69c3b7e60f4c";
+
+    SELECT TRUE INTO patientIsUnplannedAid
+    FROM person p
```

Comment:
Looks good @IipperMangesh, we can further simplify the query by removing the table `person`: it is possible to select directly from patient_program where pp.patient_id = p_patientId, and then join patient_program_attribute to check the uuid for unplanned aid